scheduler/tests.py

```diff
@@ -537,6 +537,8 @@ class TestSchedulableJob(TestBaseJob):
         # Currently ScheduledJob and RepeatableJob
         JobClass = BaseJob
         JobClassFactory = BaseJobFactory
+        RepeatableJobClassFactory = RepeatableJobFactory
```

Comment:
Should not be here for TestSchedulableJob.

Reply:
Since JobClassFactory tests both repeatable and scheduled there is no other way. I think the added test covers all the angles. Please let me know.

Reply:
`BaseTestCases.TestSchedulableJob` does not run by itself. 
It is inherited and runs in `TestSchedulableJob` and `TestRepeatableJob`.

Reply:
Oh, I see. 
What do you recommend? Scheduled and Repeatable are not the exact same animal. If we generalize, how to go about it?

Reply:
If the test should be for both, then it should be in `BaseTestCases.TestSchedulableJob` with `JobClassFactory`.
If the test is unique for one, it should be in one of the inherited classes.

Reply:
Great! I've updated the tests as per your suggestions!
Thanks!